README.md

```diff
@@ -131,6 +131,34 @@ RAILS_ENV=production bundle exec rake m2c_exist_all_storage_roots[profile]
 ```
 this will generate a log at, for example, ```log/profile_check_existence_for_all_storage_roots2017-12-11T14:25:31-flat.txt```
 
+### Run Catalog to Moab existence check for a single root and for all storage roots
+
+- To run rake tasks below, give the name of the moab storage_root (e.g. from settings/development.yml) and the last checked before date as an argument.
+
+#### Single Root
+- Without profiling
+```ruby
+RAILS_ENV=production bundle exec rake c2m_check_version_on_dir[last_checked_b4_date, fixture_sr1]
+```
+- With profiling
+```ruby
+RAILS_ENV=production bundle exec rake c2m_check_version_on_dir[last_checked_b4_date, fixture_sr1,profile]
```

Comment:
ditto on exemplar date, and do we want/need a space after the comma between `fixture_sr1` and `profile` or  must we NOT have one?

Reply:
Good catch, NO spaces!